gradle.properties

```diff
@@ -41,12 +41,12 @@ gradleFury.version=1.0.14
 
 
 android-plugin.version=7.1.3
-android-support.version=28.0.0
 # To be used for packages that depend on support lib (minSDK = 14 since 26.0.0)
 android-minSdkForSupportLib.version=14
 junit.version=4.13.2
 robolectric.version=4.7.3
 android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
Just a minor remark from https://developer.android.com/jetpack/androidx/migrate:
> Additionally, the [enableJetifier](https://developer.android.com/jetpack/androidx/migrate#migrate_an_existing_project_using_android_studio) flag mentioned in this guide can lead to slower build times and should not be used unless it’s necessary.

Reply:
I will address this!

Reply:
![Legacy Library](https://user-images.githubusercontent.com/32745243/197236757-06d5a887-1c79-4d76-8b3f-0c68e19f70c6.PNG)

This seems to be the only thing that requires the enableJetifier flag on. The logging would need to be updated as newer versions of ACRA removed `setReportSender()` which is used in `OsmApplication.java`

Reply:
Overall the build time does not seem to be severely impacted by this:
As indicated here
![Build](https://user-images.githubusercontent.com/32745243/197238320-2a1da492-2f58-4938-9b5c-0827b02640e1.PNG)

![Total time](https://user-images.githubusercontent.com/32745243/197238330-932938ed-9dbd-4528-a672-984ff2a26fd0.PNG)

Reply:
Thank you @PWRxPSYCHO for your detailed report!

If I sum up:
* the impact of `android.enableJetifier=true` looks minimal, and is _de facto_ limited to the sample app
* we may create an issue about getting rid of `android.enableJetifier=true` that would involve an upgrade of `ch.acra` and some minor refactoring around it